scripts/update-spectrum-css.js

```diff
@@ -16,7 +17,9 @@ import { readFileSync, writeFileSync } from 'fs';
 import latestVersion from 'latest-version';
 import fg from 'fast-glob';
 
-const useLatest = process.argv[2] === '--latest';
+// What tag to target, defaults to `latest`
+// Example: `node scripts/update-spectrum-css.js s2-foundations` will target the latest version of the s2-foundations tag
```

Comment:
Nice change and addition of comments! :)

Reply:
Yes! 🚀